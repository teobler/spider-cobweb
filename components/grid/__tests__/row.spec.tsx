import React from 'react';
import renderer from 'react-test-renderer';
import Row from '../row';

describe('<Row />', () => {
  it('should render a empty layout', () => {
    const layoutJson = renderer.create(<Row />).toJSON();
    expect(layoutJson).toMatchSnapshot();
  });
});
