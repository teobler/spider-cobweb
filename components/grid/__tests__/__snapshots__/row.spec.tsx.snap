// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should render a empty layout 1`] = `
<div
  className="spider-row"
  style={Object {}}
/>
`;
